scripts/validate

```diff
@@ -11,4 +11,6 @@ if [ -n "$DIRTY" ]; then
     echo Git is dirty
     git status
     exit 1
-fi
\ No newline at end of file
+fi
+
+./scripts/generate-charts
```

Comment:
@StrongMonkey anything else we need to add?

Reply:
This should not be in this PR. It needs to be a separate one.

Reply:
https://github.com/rancher/dev-charts/pull/44